org.libreoffice.LibreOffice.json

```diff
@@ -64,8 +64,8 @@
                 },
                 {
                     "type": "archive",
-                    "url": "https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.tar.xz",
-                    "sha256": "cebb705dbbe26a41d359b8be08ec066caba4e8686670070ce44bbf2b57ae113f",
+                    "url": "https://dlcdn.apache.org//ant/binaries/apache-ant-1.10.13-bin.zip",
```

Comment:
* Is the change from `archive.apache.org` to `dlcdn.apache.org` intentional or by accident?  (If intentional, a rationale in the commit message would probably be helpful.)
* The double `//` in the middle of the URL should be a single `/`.
* Is the change from `.tar.xz` to `.zip` intentional or by accident?  (If intentional, a rationale in the commit message would probably be helpful.)